Scripts/Swift/Extensions/FileManager+Utils.swift

```diff
@@ -42,4 +42,31 @@ extension FileManager {
             print("Error \(error)")
         }
     }
+
+    func replaceAllOccurrences(of original: String, to replacing: String) throws {
```

Comment:
I think we shouldn't make `throws` since it's the last catch that we can.

could we print error logs in the catch block?